frontend/src/libs/components/customer-card/styles.module.scss

```diff
@@ -0,0 +1,25 @@
+@import "src/assets/css/vars.scss";
+
+.card {
+  max-width: 540px;
+  padding: 27px 89px 39px 222px;
+  background-color: $white;
+  background-image: url("../../../assets/img/welcome-page/customer.png");
+  background-repeat: no-repeat;
+  background-size: 229px 229px;
+  border-radius: 10px;
+
+  // box-shadow: 0 2px 4px 0 rgb(173, 163, 163 / 15%);
```

Comment:
Remove comment 
Looks like you copied the styles from the Figma, It is kinda bad practice

Reply:
I looked at what parameters are specified in Figma, took the color, x, y and blur data, and used the shadow generator.
Should I not use these colors?

Reply:
I fix my error